<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSubjects</class>
 <widget class="QDialog" name="FormSubjects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_Add">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Delete">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Usu≈Ñ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_Confirm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/right-tick.png</normaloff>:/z/icons/right-tick.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Update">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/update-arrow.png</normaloff>:/z/icons/update-arrow.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Canel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/gross-dark-cross.png</normaloff>:/z/icons/gross-dark-cross.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nazwa specjalizacji</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Unikalny identyfikator spcejalizacji</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
